src/vcSettingsUI.h

```diff
@@ -3,6 +3,8 @@
 
 struct vcState;
 
+static const char* ScreenshotExportFormats[4] = { ".PNG", ".BMP", ".TGA", ".JPG" };
```

Comment:
These screenshot arrays are getting spread out over the whole program, bringing them together is going to help later